### PR TITLE
update for cedar#814

### PR DESCRIPTION
**Pull request by cdisselkoen**
*Issue #, if available:*

cedar-policy/cedar#814

CI will probably fail until cedar-policy/cedar#819 is merged.

### CODE REVIEW

File: CedarJava/src/main/java/com/cedarpolicy/model/AuthorizationResponse.java [hunk 1]
```diff
@@ -53,4 +53,14 @@ public AuthorizationResponse(
             this.warnings = ImmutableList.copyOf(warnings);
         }
     }
+
+    @Override
```
Comment (aaronjeline) [inline CedarJava/src/main/java/com/cedarpolicy/model/AuthorizationResponse.java:57]: Did you mean to include these new `toString`s?

**Review by aaronjeline — APPROVED** [review]

File: CedarJava/src/main/java/com/cedarpolicy/model/AuthorizationResponse.java (hunk 1 above)
Comment (cdisselkoen) [reply CedarJava/src/main/java/com/cedarpolicy/model/AuthorizationResponse.java:57]: Yes, they are helpful for debugging.  Debugging test failures is currently painful

**Comment by cdisselkoen** [comment]
Interesting that the CI did not in fact fail even though cedar-policy/cedar#819 has not been merged

File: CedarJava/src/main/java/com/cedarpolicy/model/AuthorizationResponse.java (hunk 1 above)
Comment (aaronjeline) [reply CedarJava/src/main/java/com/cedarpolicy/model/AuthorizationResponse.java:57]: Cool, just making sure it wasn't an accidental inclusion

**Review by khieta — APPROVED** [review]

**Comment by khieta** [comment]
Need to revert cedar#819 to fix CI 🙂

**Pull request merged**